app/views/reasons/MoreInformationView.scala.html

```diff
@@ -50,7 +50,8 @@
                 if(isSingleEntry) messages("moreInformation.single.heading") else messages("moreInformation.bulk.pageTitle"),
                 true,
                 false,
-                classes = Some("govuk-fieldset__legend")
+                classes = Some("govuk-fieldset__legend"),
+                maxCharacters = Some(1500)
```

Comment:
To maintain consistency, should we retrieve this value from MoreInformationFormProvider.maxLength?